### PR TITLE
Update Duffel Links API to use the new endpoint

### PR DESCRIPTION
**Pull request by jbernardo95**
The Duffel Links endpoint has been changed from `/links/generate_link`
to `/links/sessions`.

### CODE REVIEW

**Review by go1t — APPROVED** [review]
Thanks!

File: src/Links/Sessions.ts [hunk 1]
```diff
@@ -64,10 +64,10 @@ export class Links extends Resource {
...
 
   /**
-   * Generate a Duffel Link per the configuration
+   * Create a Duffel Links Session per the configuration
    */
-  public generateLink = async (
-    data: GenerateLinkParameters
-  ): Promise<DuffelResponse<GeneratedLink>> =>
-    this.request({ method: 'POST', path: `${this.path}/generate_link`, data })
+  public create = async (
```
Comment (go1t) [inline src/Links/Sessions.ts:69]: nit: I wonder if this should be call `createSession` for clarity 🤔

Comment (jbernardo95) [reply src/Links/Sessions.ts]: Move this to a `Sessions` class, it's now `Sessions.create`

File: src/index.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ export class Duffel {
     this.aircraft = new Aircraft(this.client)
     this.airlines = new Airlines(this.client)
     this.airports = new Airports(this.client)
-    this.links = new Links(this.client)
+    this.links = new Sessions(this.client)
```
Comment (jbernardo95) [inline src/index.ts:60]: Context on this: https://duffel.slack.com/archives/C04BCM0H42W/p1676043687668789?thread_ts=1676043331.705249&cid=C04BCM0H42W

Comment (jbernardo95) [inline src/index.ts:60]: We've decided to expose this under the `links` key to make it obvious. This is not consistent with the rest of the API. We should at some point think about making this consistent.

**Review by go1t — APPROVED** [review]

**Pull request merged**